Tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -776,6 +793,7 @@ public function testInvalidCacheConfiguration(): void
      * @param array|string $cacheConfig
      *
      * @dataProvider cacheConfigurationProvider
+     * @group legacy
```

Comment:
The legacy cases must be split into separate tests. Not everything about cache configuration is legacy.

Reply:
i don't this it's worth it

Reply:
to me it is, for 2 reasons:

- when you remove a deprecated feature, you can remove the associated legacy tests. But here, that would remove the coverage for other cache configurations too
- marking tests covering other (non-deprecated) features as `@legacy` can hide accidental usage of deprecated APIs (which is precisely the reason to make test fail on deprecations in non-legacy tests)